### PR TITLE
Fix max canvas size calculation 

### PR DESCRIPTION
**Pull request by danji90**
# How to

Use canvas-size lib for maxCanvasSize calculation to fix iOS crash

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [ ] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] The title means something for a human being.
- [x] The title contains [WIP] if it's necessary.
- [x] Labels applied. if it's a release? a hotfix?
- [ ] Tests added.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "@webcomponents/webcomponents-platform": "1.0.1",
     "@webcomponents/webcomponentsjs": "2.6.0",
     "abortcontroller-polyfill": "1.7.3",
+    "canvas-size": "^1.2.5",
```
Comment (oterral) [inline package.json:20]: Fix version pls

File: src/menus/ExportMenu/ExportMenu.js [hunk 2]
```diff
@@ -150,14 +150,16 @@ const ExportMenu = () => {
...
+        .maxArea({ usePromise: true })
+        .then((result) => {
+          const size = Math.max(result.width, result.height);
+          setMaxCanvasSize(size);
+          localStorage.setItem(LS_SIZE_KEY, size);
+        })
+        // eslint-disable-next-line no-console
+        .catch((result) => console.log('Error', result));
     }
     return () => {};
```
Comment (oterral) [inline src/menus/ExportMenu/ExportMenu.js:164]: Can you abort the calculation on unmount ?

**Review by oterral — CHANGES_REQUESTED** [review]
Nice improvment . Make sure you kill the calcualtion on unmount. and of course that it's working on edge  and mobile

File: package.json (hunk 1 above)
Comment (danji90) [reply package.json]: Pinned the version

File: src/menus/ExportMenu/ExportMenu.js (hunk 2 above)
Comment (danji90) [reply src/menus/ExportMenu/ExportMenu.js]: Using [cancelable-promise](https://www.npmjs.com/package/cancelable-promise) to cancel on unmount

**Review by oterral — APPROVED** [review]

**Pull request merged**